sdk/purview/purview-workflow-rest/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+### Features Added
+
+- Initial release of the Purview Workflow client library for JavaScript
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
```

Comment:
```suggestion

```

Reply:
I think they can be removed if no other items in it

Reply:
Fixed, thanks